### ISSUE TITLE
Some questions about accuracy predictor

### ISSUE THREAD

**Issue by gunjupark** [mit-han-lab/once-for-all#30]
Hi, I'm very interested in your works.

I want to use accuracy predictor about some other config. (like resnet based OFA ... and some)

I saw some tutorial codes about acc_predictor you uploaded, so I could understand how it look likes. 
And I saw your paper's appendix about detail of accuracy predictor.

I have a question about how much train data for training accuracy predictor.

And when you were training acc_predictor , there are ground truths you measured using whole imagenet valid set.

How many ground truth are needed?

Also, I want to know hyper parameters about accuracy predictor's training.

I hope you answer my questions, Thank you

**Comment by han-cai** [comment]
1K ~ 4K samples per resolution are usually sufficient for training the accuracy predictor. 

In the data preprocessing phase, it is important to make sure the accuracy scale is [0, 1] instead of [0, 100]. 

The optimizer is adam. The learning rate is 1e-3. The weight decay is 1e-4. This training setting works well with different batch sizes (e.g., 500, 1000, etc).

Besides, setting the bias term of the output layer as the average accuracy can improve the training stability.
